### PR TITLE
UICAT-114 Create new authority using a template by default

### PR DESCRIPTION
**Pull request by EstebanSM**
## Purpose

- [UICAT-114 Create new authority using a template by default](https://issues.folio.org/browse/UICAT-114)

In the first load of marccat, is loading a template for Bibs records, now we load a template for Auth records too.

Bibliographic template: /marccat/bibliographic-record/from-template/1
Authority template: /marccat/authority-record/from-template/1

In search we have to select Bibs or Auth record, and then we have to select New record in Actions menu.

We manage "segmentActive" to know which option is selected and manage wich api calls to do and wich options we have to show.

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by matteopascucci — APPROVED** [review]

**Comment by carolegodfrey** [comment]
Are there more details to help provide context for this PR?
When I look into Jira tickets there is no description for either of these -
https://issues.folio.org/browse/UICAT-114
and https://issues.folio.org/browse/UXPROD-2627

File: test/bigtest/interactors/action-menu.js [hunk 1]
```diff
@@ -3,22 +3,38 @@ import {
...
-  } from '@bigtest/interactor';
+    count
+} from '@bigtest/interactor';
 
-  const ActionButtonInteractor = interactor(class ButtonInteractor {
-    clickThrough = clickable();
-  });
+const ActionButtonInteractor = interactor(class ButtonInteractor {
+  clickThrough = clickable();
+});
```
Comment (carolegodfrey) [inline test/bigtest/interactors/action-menu.js:11]: Is this spacing adjustment intentional?

File: test/bigtest/tests/action-menu-test.js [hunk 2]
```diff
@@ -14,19 +14,58 @@ describe('ActionMenu', () => {
...
+    });
+
+    it('show create new auth record avaiable', () => {
+      expect(actionMenuInteractor.buttonNewBibRecord).to.be.true;
+    });
+
+  });
+
+  describe('click on Action and show aviable button for New Auth Record', function () {
+    beforeEach(async function () {
```
Comment (carolegodfrey) [inline test/bigtest/tests/action-menu-test.js:45]: Minor typo -  `aviable` vs `available`

File: test/bigtest/interactors/action-menu.js (hunk 1 above)
Comment (EstebanSM) [reply test/bigtest/interactors/action-menu.js:11]: That space was removed, in last commit doesn't exists

File: test/bigtest/tests/action-menu-test.js (hunk 2 above)
Comment (EstebanSM) [reply test/bigtest/tests/action-menu-test.js:45]: i have solved the typo mistake

File: test/bigtest/tests/action-menu-test.js [hunk 3]
```diff
@@ -36,14 +75,26 @@ describe('ActionMenu', () => {
 
   });
 
-  describe('click on Action and show aviable button for New Bib Record', function () {
+  describe('click on Action and show available button for New Bib Record', function () {
```
[the remark this answers is not in the document]
Comment (EstebanSM) [reply test/bigtest/tests/action-menu-test.js:78]: **i have included a desciption in Jira and PR**

In the first load of marccat, is loading a template for Bibs records, now we load a template for Auth records too.

+Bibliographic template:+ /marccat/bibliographic-record/from-template/1
+Authority template+: /marccat/authority-record/from-template/1

In search we have to select Bibs or Auth record, and then we have to select New record in Actions menu.

We manage "segmentActive" to know which option is selected and manage wich api calls to do and wich options we have to show.

File: src/components/Search/Panel/SearchPanel.js [hunk 4]
```diff
@@ -28,7 +28,7 @@ import { ACTION } from '../../../redux/actions/Actions';
 import { findYourQuery } from '../Filter';
 import { remapFilters, findParam } from '../../../shared';
 import { EMPTY_STRING, SEARCH_SEGMENT } from '../../../config/constants';
-import { resetFilterSearch, historySearchAction, searchDetailAction } from '../Actions';
+import { resetFilterSearch, segmentActive, historySearchAction, searchDetailAction } from '../Actions';
```
[the remark this answers is not in the document]
Comment (EstebanSM) [reply src/components/Search/Panel/SearchPanel.js:31]: Hi Carole, i have included a desciption in Jira and PR

[the remark this answers is not in the document]
Comment (carolegodfrey) [reply src/components/Search/Panel/SearchPanel.js:31]: Thank You @EstebanSM -- really helpful to have this context to understand these changes

**Review by carolegodfrey — APPROVED** [review]

File: src/components/Search/Panel/SearchPanel.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (EstebanSM) [reply src/components/Search/Panel/SearchPanel.js:31]: Thank you @carolegodfrey 
I have learned that in next time i have to explain the purpose and context for a better revision of the code.

**Pull request merged**